var/spack/repos/builtin/packages/qualimap/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Qualimap(Package):
+    """Qualimap 2 is a platform-independent application written in Java
+    and R that provides both a Graphical User Inteface (GUI) and a
+    command-line interface to facilitate the quality control of alignment
+    sequencing data and its derivatives like feature counts."""
+
+    homepage = "http://qualimap.conesalab.org/"
+    url      = "https://bitbucket.org/kokonech/qualimap/downloads/qualimap_v2.2.1.zip"
+
+    version('2.2.1', sha256='08f1d66e49c83c76c56c4225c53aee44f41e0592c8bdc84b8c4ecd975700e045')
+
+    depends_on('openjdk', type=('build', 'run'))
```

Comment:
```yaml
grep -r java etc/spack/
etc/spack/defaults/packages.yaml:      java: [openjdk, jdk, ibm-java]
```
Does it only run with OpenJDK or should it run also with Sun's JDK and IBM Java? If so, you can approve this suggestion here. As you see, in spack, `java` is virtual dependency, where `openjdk` is the default provider.

Please note: The different providers `java` provide different version ranges of Java and new versions could be added at any time, and the default version can change as well, so please, if you know, add the version range as well.

To you know the version range of Java it works with? Then you could add the version range as well, e.g:
```py
depends_on('java@8:', type='run')
```
'build' does not apply because it's just 'run', you just install the binary download here,
and you can find exactly this line on other java packages in spack:
```yaml
grep -r depends_on.*java var/spack/repos/builtin
```
If you'd agree to be notified by spackbot when someone submits a PR for qualimap, you can (would be nice to) follow the suggestion of spackbot and add the maintainers line with your gh handle as well.
```suggestion
    depends_on('java', type='run')
```